students/Isabella_Kemp/lesson02/FizzBuzz-exercise.py

```diff
@@ -0,0 +1,15 @@
+#Isabella Kemp
+#12/23/2019
+#Fizz Buzz
+
+for x in range (1,101):
```

Comment:
You'll want to remove the space after range, and add one after the comma.  So: for x in range(1, 101):